OpenRA.Mods.RA/OpenRA.Mods.RA.csproj

```diff
@@ -80,7 +81,7 @@
     <Compile Include="Activities\Infiltrate.cs" />
     <Compile Include="Activities\LayMines.cs" />
     <Compile Include="Activities\Leap.cs" />
-    <Compile Include="Activities\RAHarvesterDockSequence.cs" />
+    <Compile Include="Activities\HarvesterDockSequence.cs" />
```

Comment:
I'll need to look at that before answering. At the time of doing this (the first time) it was impossible. If it requires moving any more classes now I'll leave it for another PR.